[pytest]
markers =
    no_catch_loop_exceptions: no catch unhandled exceptions from event loop
    allow_get_event_loop: allow to call asyncio.get_event_loop()
